group_project/assignment_4.py

```diff
@@ -1,5 +1,7 @@
 def how_many_times(a_string, a_word):
-    pass
+    if a_word in a_string:
+        return a_string.count(a_word)
+    return 0
 
```

Comment:
You do not need the if statement. You can simply return a_string.count(a_word).

Reply:
But what if the a_word is not in a_string ?

Reply:
It returns a zero automatically if the word is not in it if you do not use an if statement. If you do the If statement route, it will return None so you do have to specify return 0.

def how_many_times(a_string, a_word):
    return a_string.count(a_word)

Reply:
Ah cool! thanks for explaining that.